### PR TITLE
Merged code for post and attachment

### PR DESCRIPTION
**Pull request by joshpham97**

### CODE REVIEW

File: src/main/webapp/templates/postTemplate.jsp [hunk 1]
```diff
@@ -25,13 +25,14 @@
                 <%
                     }
                 %>
-                <a href="/AttachmentServlet"><i class="fas fa-edit mr-2"></i></a>
-                <a href="/AttachmentServlet"><i class="fas fa-trash mr-2"></i></a>
+                <a href="PostEditServlet?postId=<%= post.get("postID") %>"><i class="fas fa-edit mr-2"></i></a>
+                <a href="PostServlet?action=delete&postID=<%= post.get("postID") %>"><i class="fas fa-trash mr-2"></i></a>
             </div>
         </div>
```
Comment (ExZos) [inline src/main/webapp/templates/postTemplate.jsp:31]: It doesn't show up in the changes i just highlighted this but
can you also add the download attachment link for the attachment icon?

File: src/main/java/server/dabatase/daoimpl/PostDAO.java [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            ps.setDate(3, date);
+            ps.setString(4, message);
+            ps.setInt(5, postId);
+            ps.executeUpdate();
+
+            post.setPostID(postId);
+            post.setUsername(uname);
+            post.setTitle(title);
+            post.setMessage(message);
+            post.setDateModified(date.toLocalDate().atStartOfDay());
```
Comment (ExZos) [inline src/main/java/server/dabatase/daoimpl/PostDAO.java:180]: Update sets the date_modified to the start of the current date
I think its from when we were using LocalDate so you just switch to LocalDateTime

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -38,7 +38,7 @@
                         <a class="mb-0 nav-link active" href="index.jsp"><i class="fas fa-home mr-2"></i>Home</a>
                     </li>
                     <li class="nav-item">
-                        <a class="mb-0 nav-link" href="post.jsp"><i class="fas fa-plus mr-2"></i>New Post</a>
+                        <a class="mb-0 nav-link" href="edit.jsp"><i class="fas fa-plus mr-2"></i>New Post</a>
```
Comment (ExZos) [inline src/main/webapp/index.jsp:41]: This linking doesn't seem to work for me. Maybe the wrong link?

File: src/main/java/server/dabatase/daoimpl/PostDAO.java (hunk 2 above)
Comment (ExZos) [reply src/main/java/server/dabatase/daoimpl/PostDAO.java:180]: I think this also applies to post message

File: src/main/webapp/templates/postTemplate.jsp (hunk 1 above)
Comment (joshpham97) [reply src/main/webapp/templates/postTemplate.jsp:31]: Attachment download linked added :)

File: src/main/java/server/dabatase/daoimpl/PostDAO.java (hunk 2 above)
Comment (joshpham97) [reply src/main/java/server/dabatase/daoimpl/PostDAO.java:180]: Yes we should use LocalDateTime instead

File: src/main/webapp/index.jsp (hunk 3 above)
Comment (joshpham97) [reply src/main/webapp/index.jsp:41]: Yeah you are supposed to call PostEditServlet to load the post first before redirecting the user to edit.jsp, will fix that

**Pull request merged**